service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/helpers/Tools.java

```diff
@@ -44,7 +45,7 @@
             return Observable.just(result);
         }
         catch (JsonProcessingException e) {
-            return Observable.error(e);
+            return Observable.error(new IotHubException("Failed to parse the resonse"));
```

Comment:
Is this what's decided to throw as an error to user?

Reply:
yes this is what we discussed.